fetch.js

```diff
@@ -73,10 +73,16 @@ function checkForHttpErrors (fetchRes) {
 function parseResponse (fetchRes, body) {
   // check for error code
   if (fetchRes.status !== 200) {
-    throw rpcErrors.internal(`Non-200 status code: '${fetchRes.status}'`, body)
+    throw ethErrors.rpc.internal({
+      message: `Non-200 status code: '${fetchRes.status}'`,
+      data: body,
+    })
   }
   // check for rpc error
-  if (body.error) throw rpcErrors.internal(body.error.toString(), body.error)
+  if (body.error) throw ethErrors.rpc.internal({
+    message: body.error.toString(),
```

Comment:
This parameter should be dropped (see #20), but this can be done in a separate PR.